tests/test_packages/test_connections/test_ledger_api/utils.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This module contains some utilities for the ledger API connection tests."""
+from typing import Dict, Optional
+
+from aea.crypto.base import Crypto
+from aea.crypto.ethereum import DEFAULT_GAS_PRICE, EthereumApi, GAS_ID
+from aea.mail.base import Address
+
+
+def make_ethereum_transaction(
```

Comment:
adapted from `EthereumApi.transfer`

Reply:
Use the ledger_api and crypto modules instead